index.html

```diff
@@ -159,13 +157,32 @@ <h4 class="subtitle is-5">Make a pull request & add your github account</h4>
 
     <!-- Contact -->
   <section class="section" id="contact">
-    <div class="container">
+    <div class="columns">
       <div class="section-contact-heading ">
-        <h3 class="title is-2">Contact</h3>
+        <h3 class="title is-2">Contact our teams :</h3>
         <h4 class="subtitle is-5">Get in touch</h4>
       </div>
-
-      <div class="social-media-links">
+ <div class="form">
+<form
+  action="https://formspree.io/mlepnabn"
+  method="POST"
+>
+<br><br><br><br><br>
```

Comment:
katste3mel bezaf had <br> keep the code clean as possible . (f bluma kayen bezaf dial les class li ghadi ydirou l résultat li baghi twsselou khassek ghi t9ra chwiya f documentation dialhoum & i'm sure ghadi t9der tgadhoum)